lib/controllers/hosted-skill-controller/helper.js

```diff
@@ -81,3 +95,64 @@ function handleSkillStatus(response, skillId, callback) {
         callback(`${errMessage}${errReason}Infrastructure provision for the hosted skill failed. Please try again.`);
     }
 }
+
+function downloadAuthInfoScript(callback) {
+    const authInfoUrl = CONSTANTS.HOSTED_SKILL.S3_SCRIPTS.AUTH_INFO.URL;
+    const authInfoPath = path.join(os.homedir(),
+        CONSTANTS.FILE_PATH.ASK.HIDDEN_FOLDER,
+        CONSTANTS.FILE_PATH.ASK.AUTH_INFO);
+    this.downloadScriptFromS3(authInfoUrl, authInfoPath, (err) => {
```

Comment:
I think you can do similar to function below

this.downloadScriptFromS3(authInfoUrl, authInfoPath, (err) => callback(err || null));

Reply:
Updated!